packages/wdio-appium-service/src/launcher.js

```diff
@@ -0,0 +1,116 @@
+import logger from '@wdio/logger'
+import { createWriteStream, ensureFileSync } from 'fs-extra'
+import { spawn } from 'child_process'
+import { promisify } from 'util'
+import getFilePath from './utils/getFilePath'
+
+const log = logger('@wdio/appium-service')
+const DEFAULT_LOG_FILENAME = 'appium.txt'
+
+export class AppiumLauncher {
+    constructor() {
+        this.logPath = null
+        this.command = ''
+        this.appiumArgs = []
+        this.waitStartTime = 0
+    }
+
+    async onPrepare(config) {
+        const appiumConfig = config.appium || {}
+
+        this.logPath = appiumConfig.logPath
+        this.command = appiumConfig.command || this._getAppiumCommand()
```

Comment:
shouldn't we use the Appium binary that has been downloaded as dependency (under `/packages/wdio-appium-service/node_modules/.bin/appium`?

Reply:
It now uses the path to the Appium binary in `node_modules`